orchestration/ptycho_jobscript.py

```diff
@@ -0,0 +1,169 @@
+import os
```

Comment:
I would like to move this file into flows/bl7012, but while it's a dependency to NERSC that will be really tough. Future thought.